packages/pipeline-services/src/validation/index.test.ts

```diff
@@ -146,65 +146,6 @@ describe("getNodeProblems", () => {
     expect(problems[0].info.property).toBe("filename");
   });
 
-  it("should not have issues when required property has a default value", () => {
```

Comment:
I think we should update this test case to mark it as an error if the pipeline has an "initial" value and then the value is deleted, it should have a validation error if required

Reply:
Does what I just pushed look okay?